console-subscriber/tests/support/task.rs

```diff
@@ -122,6 +110,7 @@ impl ExpectedTask {
 
     /// Returns an error specifying that no match was found for this expected
     /// task.
+    #[allow(clippy::result_large_err)]
```

Comment:
nit, take out or leave it: maybe we should allow this globally for the whole `impl` block?

Reply:
Make sense. Moved.